app/questionnaire/questionnaire_schema.py

```diff
@@ -336,12 +343,32 @@ def _update_answer_dependencies_for_value_source(
                     self._answer_dependencies_map[answer_id_for_block] |= {
                         self._get_answer_dependent_for_block_id(block_id=block_id, answer_id=answer_id)  # type: ignore
```

Comment:
Add #type: ignore reason above usage (also on unchanged line 333)

Reply:
Could also add type hinting and pipe syntax to this to method's declaration while here

Reply:
That's not something I've added so I didn't really check what is the reason but can look into it when working on the one below.